sublime_linter.py

```diff
@@ -169,6 +169,15 @@ def on_hover(self, view, point, hover_zone):
             if persist.settings.get('show_hover_region_report'):
                 SublimeLinter.shared_plugin().open_tooltip(view, point, True)
 
+    def on_selection_modified_async(self, view):
```

Comment:
I would really appreciate if we don't immediately start and fill sublime_linter.py with stuff that belongs to a special view? I've shown with the statusbar view and the busy indicator, that this really helps readability. 

I think this is really important especially for open source projects, where a user must find a bug and a way to help and contribute very fast, not knowing the code base.